src/components/Spawn/index.tsx

```diff
@@ -1,9 +1,11 @@
 import React from "react";
```

Comment:
this file includes shared components between the spawn host and spawn volume pages